brainscore/benchmarks/Lee2020.py

```diff
@@ -0,0 +1,181 @@
+            URL = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116},
+            eprint = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116.full.pdf},
+            journal = {bioRxiv}}'''
+
+class DicarloLee2020ITTissueResponseCorrelation(BenchmarkBase):
+
+    def __init__(self, metric=ks_similarity, bootstrap_samples=100_000, num_sample_arrays_candidate=10, ceiler_fn=InternalConsistency):
+
+        def load_target_assembly():
+            assembly = make_static(brainscore.get_assembly('dicarlo.MajajHong2015').sel(region='IT'))
+            return assembly[:, np.in1d(assembly.presentation.image_id, list(set(stimulus_set['image_id'])))]
+
+
+        stimulus_set = Rajalingham2019()._target_assembly.stimulus_set
+        assembly = load_target_assembly()
+
+        super(self).__init__(identifier='dicarlo.Lee2020.IT-tissue',
+                             ceiling_func=lambda: ceiler_fn()(assembly.transpose('presentation','neuroid')),
+                             version=0.1,
+                             parent='IT',
+                             bibtex=BIBTEX)
+
+        self._stimulus_set = stimulus_set
+        self._target_assembly = average_repetition(assembly)
+        self._target_statistic_fn = self.create_tissue_response_correlation_target
+        self._metric = metric
+
+        self.bootstrap_samples = bootstrap_samples
+        self.num_sample_arrs = num_sample_arrays_candidate
+        self._arr_size_mm = (np.ptp(self._target_assembly.neuroid.x.data),

```

Comment:
you might just want to write out `array`, you don't save that many characters by removing `ay` ;-)